src/keys.rs

```diff
@@ -7,9 +7,21 @@
 use glib::translate::*;
 use ffi;
 use libc::c_uint;
+use std::mem::transmute;
 
 pub fn keyval_name(keyval: u32) -> Option<String> {
     unsafe {
         from_glib_none(ffi::gdk_keyval_name(keyval as c_uint))
     }
 }
+
+pub fn keyval_to_unicode(keyval: u32) -> Option<char> {
+	unsafe {
```

Comment:
There still seems to be a problem with the indentation -- six spaces instead of four.